image/deployer.Dockerfile

```
@@ -0,0 +1,9 @@
+FROM golang:1.15-alpine AS build
+COPY ./ /app
+WORKDIR /app
+RUN CGO_ENABLED=0 go build -ldflags='-s -w' -o ./deployer ./cmd/deployer
+
+FROM scratch
```

```suggestion
FROM alpine:3.12
```

WDYT about using alpine so it is easy to `docker exec` into for debugging purpose?

---

```suggestion
FROM scratch
RUN apk add --no-cache docker-cli
```

WDYT about adding the Docker CLI (in Alpine) to allow local access for debugging?